server.js

```diff
@@ -598,8 +605,13 @@ proxy.on('proxyReq', function(proxyReq, req, res) {
 /**
  * Redirect to add the trailing slash without which mysterious bugs occur...
  */
-app.get(pathPrefix + '_design/' + db.settings.ddoc + '/_rewrite', function(req, res) {
-  res.redirect(appPrefix);
+[
+  appPrefix,
+  pathPrefix
+].forEach(function(url) {
+  app.get(url.substring(url.length - 1), function(req, res) {
```

Comment:
I'm not completely comfortable with this. It:
 - Presumes that anything added to that list has a `/` at the end (which for now it does)
 - And then uses that knowledge to remove it from the end

I like it more than what was there before though, where we effectively define the variable twice, and opposite ends of the file, and have one have a trailing space and the other not.